ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -24,6 +24,8 @@ type ssh_kbdinteractivedevices = string with match (SELF, "^(bsdauth|pam|skey)$"
 type ssh_kexalgorithms = string with match (SELF, "^(diffie-hellman-group-exchange-sha256|" +
     "ecdh-sha2-nistp(256|384|521)|curve25519-sha256@libssh.org)$");
 type ssh_MACs = string with is_valid_ssh_MAC(SELF);
+type ssh_gssapikexalgorithms = string with match (SELF, "^(gss-gex-sha1-|gss-group1-sha1-|" +
+    "gss-group14-sha1-|gss-group14-sha256-|gss-group16-sha512-|gss-nistp256-sha256-|gss-curve25519-sha256-)$");
```

Comment:
Thanks, `choice()` is the more modern solution for this, but this fits in with the rest of the schema so it's fine.